FFMpegCore/FFMPEG/Argument/Atoms/VideoCodecArgument.cs

```diff
@@ -5,23 +5,25 @@ namespace FFMpegCore.FFMPEG.Argument
     /// <summary>
     /// Represents video codec parameter
     /// </summary>
-    public class VideoCodecArgument : Argument<VideoCodec>
+    public class VideoCodecArgument : Argument<string>
```

Comment:
Wouldn't it be a better solution to extend the enum with new members instead?

Reply:
As i mensioned above it is not the best solution